src/components/ResourceList/ResourceList.tsx

```diff
@@ -400,11 +400,7 @@ export const ResourceList: ResourceListType = function ResourceList<ItemType>({
   const renderItemWithId = (item: ItemType, index: number) => {
     const id = idForItem(item, index);
 
-    return (
-      <li key={id} className={styles.ItemWrapper}>
-        {renderItem(item, id, index)}
-      </li>
-    );
+    return renderItem(item, id, index);
   };
```

Comment:
Interested to talk about this. It could be really dangerous for accessibility, but I feel like removing the wrapper from the parent component enables the focus `z-index` to be managed without passing state between components.

Reply:
Dangerous as in if someone uses the resource list but not the resource list item and rolls their own?

Reply:
Yeah exactly, any custom `ResourceListItem` needs to be wrapped in a `li`.

Reply:
I think this will be ok for most cases, but I do worry a bit about it too. Even if someone uses the `ResourceItem` but wraps it in a div or other non list element, they would break the structure of the list. Doing a quick poke around in web I don't see anyone doing that though. Do we always recommend to only render a `ResourceItem` from renderItem?

Reply:
Looking at the example files we do that ^

Reply:
What about if we thre a warning if the returned component isn't a ResourceItem?

Reply:
Yeah that would make me feel a little bit better. Do we have any examples of this?

Reply:
We have a utility function `isElementOfType` that could be used to check against.

Reply:
@dfmcphee @kyledurand I have implemented the warning would appreciate some fresh eyes 👁️